lib/js/components/Tile/Tile.consts.ts

```diff
@@ -46,3 +46,12 @@ export const TILE_ADDITIONAL_TEXT_MAX_WIDTHS = {
 } as const;
 
 export type TileAdditionalTextMaxWidth = Value<typeof TILE_ADDITIONAL_TEXT_MAX_WIDTHS>;
+
+export const TILE_COMPACT_LAYOUTS = {
```

Comment:
why the name `TILE_COMPACT_LAYOUTS` - if I understand correctly, this is the list of all layouts (because `default` is present on this list), not only the compact ones.

Reply:
discussed offline - we are leaving the names as they are.